docker-compose.test.yml

```diff
@@ -12,6 +12,10 @@ services:
     environment:
       DFE_TEACHERS_PAYMENT_SERVICE_DATABASE_USERNAME: postgres
       DFE_TEACHERS_PAYMENT_SERVICE_DATABASE_HOST: db
+      DFE_SIGN_IN_API_CLIENT_ID: teacherpayments
+      DFE_SIGN_IN_API_SECRET: secret
+      DFE_SIGN_IN_API_ENDPOINT: https://example.com
+      DFE_SIGN_IN_ROLE: Access to Teacher Payments
```

Comment:
There's also an `.env.test` file where these environment variables can be added. That way the build won't fail in non-docker environments.